### ISSUE TITLE
Viewcode fails parsing Python 3.5 matrix multiplication operator '@'

### ISSUE THREAD

**Issue by nbedou** [sphinx-doc/sphinx#2479]
Using:
-  latest Sphinx dev version
-  latest pygments dev version
-  Python 3.5

I obtained the following error when parsing a Python 3 module using the multiplication operator '@':

```
File "/home/nicolas/anaconda3/envs/bugviewcode/lib/python3.5/site-packages/pygments/filters/__init__.py", line 196, in filter
raise self.exception(value)
 pygments.filters.ErrorToken: @
```

Remark: pygments used with command-line tool pygmentize can parse the file.

More details on the following test repo: https://github.com/nbedou/bugviewcode

**Comment by Rutzmoser** [comment]
I have the same issue. It does not show up with Sphinx 1.3.1 but all subsequent releases.

**Comment by twmacro** [comment]
I get this too. I rolled sphinx back to 1.3.1 ... but is there a work-around for later versions?

**Comment by neuralyzer** [comment]
I had the same issue. I could resolve it by adding the line

```
highlight_language = "python3"
```

to the conf.py file.

**Comment by twmacro** [comment]
That didn't work for me.

**Comment by tk0miya** [comment]
Sorry for inconvenience. I fixed this at 04723c1.
Thank you for reporting.
